src/containers/App.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook, removeBook, changeFilter } from '../actions';
+import BooksList from '../components/BooksList.js';
+import BooksForm from '../components/BooksForm.js';
+
+const mapDispatchToProps = (dispatch) => ({
+  createBook: (book) => dispatch(createBook(book)),
+  removeBook: (book) => dispatch(removeBook(book)),
+  changeFilter: (filter) => dispatch(changeFilter(filter)),
+});
+
+const mapStateToProps = (state) => ({
+  filter: state.filter,
+  books: state.books,
+});
```

Comment:
Hi guys, doing this really defeats the purpose of having redux. we are supposed to connect directly to it from the component that needs the data or needs to make that dispatch. this was clearly states in milestone 2 for `BookList` to connect to the store. since its a past milestone i'm marking it optional here but that is the right way to handle data.
[OPTIONAL]